cli/cmd/org/edit.go

```diff
@@ -52,22 +52,18 @@ func EditCmd(cfg *config.Config) *cobra.Command {
 			}
 
 			org := resp.Organization
-
-			// Set the default values for edit flags with current values for org
-			cmd.Flag("description").DefValue = org.Description
-
+			req := &adminv1.UpdateOrganizationRequest{
+				Name: org.Name,
+			}
 			if cfg.Interactive {
-				err = cmdutil.SetFlagsByInputPrompts(*cmd, "description")
+				description, err = cmdutil.InputPrompt("Enter the description", org.Description)
 				if err != nil {
 					return err
 				}
+				req.Description = &description
```

Comment:
If `description` is passed via flag and `!cfg.Interactive`, description will not be updated